<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temp</class>
 <widget class="QWidget" name="temp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytTemp">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytTemp
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytTemp.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmmndBtn">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>10</y>
     <width>151</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/cmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sttsBtn">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>10</y>
     <width>151</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/sttsBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnBtn">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>10</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstBtn">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>10</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingBtn">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clrBtn">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/clrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pwrBtn">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pwrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pwrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckBtn">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kab1TmpLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab1TmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kab2TmpLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab2TmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mbnDrvTmpLbl">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnDrvTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mbsDrvTmpLbl">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbsDrvTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ctrlBoxTmpLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#ctrlBoxTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outTmpLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#outTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="snrfDrvTmpLbl">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#snrfDrvTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="powPckTmpLbl">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#powPckTmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kab1HmdLbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab1HmdLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kab2HmdLbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab2HmdLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ctrlBoxHmdLbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#ctrlBoxHmdLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outHmdLbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#outHmdLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
